stories/LinkPointer.stories.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf'
+  import Link from '@/ui/LinkPointer.svelte'
+</script>
+
+<Meta title="Example/LinkPointer" />
+
+<Story name="LinkPointer">
+  <div class="body-3">
+    <a href="/" class="link-pointer">body-3</a>
+    <br />
+    <Link href="/" label="body-3 + pointer" />
```

Comment:
Watch this in the browser:
hover line starts thin and at the end becomes thick

Reply:
Start:
![image](https://user-images.githubusercontent.com/6568353/190456238-6066617e-e6e2-48a9-90ee-c9491bc65112.png)

End:
![image](https://user-images.githubusercontent.com/6568353/190456319-f99358cb-bc90-4274-8fe6-9169f3e7be6f.png)

Reply:
Tested in browserstack on multiple browsers/platforms. Looks like a local issue. Make sure there is no zoom or other rendering changes